src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -3,20 +3,23 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class Drive {
+public class Drive extends SubsystemBase {
 
   public Encoder leftEncoder;
   public Encoder rightEncoder;
 
   private Gearbox leftGearbox;
   private Gearbox rightGearbox;
   // left gear box CAN ids
-  private final int LEFT_BACK_CAN_ID = 6;
-  private final int LEFT_FRONT_CAN_ID = 12;
+  private final int LEFT_BACK_CAN_ID = 6;//6
+  private final int LEFT_FRONT_CAN_ID = 5;//12
```

Comment:
Left is now:

  private final int LEFT_BACK_CAN_ID = 11;
  private final int LEFT_FRONT_CAN_ID = 12;
  //  private final int LEFT_MIDDLE_CAN_ID = 13;